css-typed-om/Overview.bs

```diff
@@ -2903,6 +2903,49 @@ depending on the property they came from:
         return "currentcolor".
     2. Otherwise, return the result of serializing the <<color>> value.
 
+: 'stroke-align'
+::
+    1. Return the string representation of the keyword:
+        * center,
+        * inset,
+        * outset.
+
+: 'stroke-break'
+::
+    1. Return the string representation of the keyword:
+        * bounding-box,
+        * slice,
+        * clone.
+
+: 'stroke-linecap'
+::
+    1. Return the string representation of the keyword:
+        * butt,
+        * round,
+        * square.
+
+: 'stroke-linejoin'
+::
+    1. Let |values| initially be the empty [=list=].
+    2. If 'stroke-linejoin' has one of the following keywords,
+        append the string representation of the keyword to |values|:
+        * crop,
+        * arcs,
+        * miter.
+    3. If 'stroke-linejoin' has one of the following keywords,
+        append the string representation of the keyword to |values|:
+        * bevel,
+        * round,
+        * stupid.
+    4. Return the result of concatenating all the items in |values|,
```

Comment:
It might be empty, need to handle that case too (emit the shortest value that triggers the initial-value behavior; check browsers to see what they do).